validator/keymanager/v2/direct/import.go

```diff
@@ -39,6 +39,15 @@ func (dr *Keymanager) ImportKeystores(
 			return errors.Wrap(err, "could not add to progress bar")
 		}
 	}
+	foundKey := map[string]bool{}
+	var strKey string
+	for j := range pubKeys {
+		strKey = hex.EncodeToString(pubKeys[j])
```

Comment:
Only used within this block

```suggestion
		strKey := hex.EncodeToString(pubKeys[j])
```